.github/workflows/copy-images-to-dockerhub.yaml

```diff
@@ -11,23 +11,27 @@ permissions:
   id-token: write #required for GCP Workload Identity federation
 
 jobs:
-  copy-images:
-    strategy:
-      fail-fast: false
-      matrix:
-        IMAGE_NAME: [validator, forge, tools, faucet, indexer, node-checker]
-
+  wait-for-images-to-have-been-build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-
       - name: Wait for images to have been built
-        timeout-minutes: 20
+        timeout-minutes: 30
         uses: lewagon/wait-on-check-action@v1.0.0
         with:
           ref: ${{ github.ref }}
           check-regexp: "rust-images.*"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30 # wait 30 seconds between making polling API calls, default is 10 but we ran in the past into rate-limiting issues with too frequent polling
```

Comment:
but we ran into rate-limiting issues in the past due to too-frequent polling

Reply:
oh well, I increased this timeout since sometimes our docker builds take longer than 20 minutes these days and then the copy-images job fails. I know this effectively means we are polling longer, but I'm not too concerned with this considering that my other change reduces the requests/s by factor 18x